docs/tutorials/basic/data.md

```diff
@@ -1,120 +1,34 @@
-# Iterators - Loading data
 
-This tutorial we focus on how to feeding data into a training and inference
-program. Most training and inference modules in MXNet accepts data iterators,
-which simplifies this procedure, especially when reading large datasets from
-filesystems. Here we discuss the API conventions and several provided iterators.
+In this tutorial, we will show how to feed data into *MXNet* training and inference programs using Data Iterator, the abstraction provided by MXNet, brief about some of the Iterators provided by MXNet: `NDArrayIter`, `CSVIter`, `ImageIter`, how to create a Custom Operator, RecordIO and ImageIO.
 
-Data iterators in MXNet is similar to the iterator in Python. In Python, we can
-use the built-in function `iter` with an iterable object (such as list) to
-return an iterator. For example, in `x = iter([1, 2, 3])` we obtain an iterator
-on the list `[1,2,3]`. If we repeatedly call `x.next()` (`__next__()` for Python
-3), then we will get elements from the list one by one, and end with a
-`StopIteration` exception.
+## MXNet Data Iterator  
+Data Iterators in *MXNet* are similar to the built-in function `iter` in ``Python``. In ``Python`` `iter` allows to fetch items sequentially by calling  `next()` on __iterable__ collection objects such as a Python `list`. `iter` provides a abstract interface for traverising various types of __iterable__ collections without needing to expose the underlying data structure.  
 
-## Introduction
+In MXNet, __Iterators__ return a batch of data `DataBatch` on each to call `next`.
```

Comment:
__Iterators__, Data Iterator, __iterable__, etc are confusing.
consider use the class name DataIter.